src/jsMain/kotlin/baaahs/SheepSimulator.kt

```diff
@@ -22,7 +27,8 @@ class SheepSimulator {
     private val sheepModel = SheepModel().apply { load() }
     private val shows = AllShows.allShows
     private val visualizer = Visualizer(sheepModel, display.forVisualizer())
-    private val pinky = Pinky(sheepModel, shows, network, dmxUniverse, display.forPinky())
+    private val beatSource: BeatSource = BridgedBeatSource("${window.location.hostname}:${Ports.SIMULATOR_BRIDGE_TCP}")
```

Comment:
Looks like we get a websockets error if the bridge isn't running:
`SheepSimulator.kt:114 WebSocket connection to 'ws://127.0.0.1:8005/bridge/beatSource' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED`

Maybe we could swap in a simulated beat source (like just 120bpm or whatever) either if the bridge isn't up, or based on some other config, so the simulator keeps showing thumpies?

Reply:
Totally, but does this only affect the simulator?

Reply:
If we're not getting beat data in production, I'm thinking we probably shouldn't fabricate it, right? But for simulator, fake data is cool.